doc/index.md

```diff
@@ -4,194 +4,149 @@ icon: /assets/image/espnet.png
+        icon: fa-solid:server
+        link: ./espnet2_tutorial.md
+
+  - header: Comprehensive Task Coverage
+    description: We offer complete recipes for a wide range of speech processing tasks.
     bgImage: https://theme-hope-assets.vuejs.press/bg/2-light.svg
-    bgImageDark: https://theme-hope-assets.vuejs.press/bg/2-dark.svg
     bgImageStyle:
       background-repeat: repeat
       background-size: initial
     features:
       - title: "ASR: Automatic Speech Recognition"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#asr-automatic-speech-recognition
+        link: ./recipe/asr1.md
+        icon: material-symbols-light:speech-to-text-rounded
 
       - title: "TTS: Text-to-speech"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#tts-text-to-speech
+        link: ./recipe/tts1.md
+        icon: material-symbols-light:text-to-speech-rounded
+
+      - title: "Speech Enhancement"
+        link: ./recipe/enh1.md
+        icon: material-symbols:adaptive-audio-mic-rounded
+
+      - title: "Weakly-supervised Learning"
+        link: ./recipe/s2t1.md
+        icon: guidance:children-must-be-supervised
+
+      - title: "Speaker Embedding"
+        link: ./recipe/spk1.md
+        icon: ic:sharp-safety-divider
+
+      # - title: "SSL: Self-supervised Learning"
+      #   link: ./recipe/ssl1.md
 
-      - title: "SE: Speech enhancement (and separation)"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#se-speech-enhancement-and-separation
+      # - title: "Speech Language Model"
+      #   link: ./recipe/speechlm1.md
 
-      - title: "SUM: Speech Summarization"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#sum-speech-summarization
+      # - title: "SLU: Spoken Language Understanding"
+      #   link: ./recipe/slu1.md
 
-      - title: "SVS: Singing Voice Synthesis"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#svs-singing-voice-synthesis
+      - title: "Speech-to-Text Translation"
+        link: ./recipe/st1.md
+        icon: ph:translate
 
-      - title: "SSL: Self-supervised Learning"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#ssl-self-supervised-learning
+      # - title: "Speech-to-Speech Translation"
+      #   link: ./recipe/s2st1.md
 
-      - title: "UASR: Unsupervised ASR "
-        details: "EURO: ESPnet Unsupervised Recognition - Open-source"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#uasr-unsupervised-asr-euro-espnet-unsupervised-recognition---open-source
+      - title: "Speech Codec"
+        link: ./recipe/codec1.md
+        icon: material-symbols:hd-outline
 
-      - title: "S2T: Speech-to-text with Whisper-style multilingual   multitask models"
-        link: https://github.com/espnet/espnet?tab=readme-ov-file#s2t-speech-to-text-with-whisper-style-multilingual-multitask-models
+      - title: "... And much more!"
+        link: recipe/
```

Comment:
Are they reserved for next PR? I saw a few tutorials are ready but not linked here

Reply:
Good point, I also added other recipes too on 2d8ea77! Thanks for pointing out!